cmd/migration/v0.9.1-v1/mapper/trade.go

```diff
@@ -0,0 +1,84 @@
+package mapper
+
+import (
+	v091domain "github.com/tdex-network/tdex-daemon/cmd/migration/v0.9.1-v1/v091-domain"
+	"github.com/tdex-network/tdex-daemon/internal/core/domain"
+)
+
+func (m *mapperService) FromV091TradesToV1Trades(
+	trades []*v091domain.Trade,
+) ([]*domain.Trade, error) {
+	res := make([]*domain.Trade, 0, len(trades))
+	for _, v := range trades {
+		trade, err := m.fromV091TradeToV1Trade(v)
+		if err != nil {
+			return nil, err
+		}
+		res = append(res, trade)
+	}
+
+	return res, nil
+}
+
+func (m *mapperService) fromV091TradeToV1Trade(
+	trade *v091domain.Trade,
+) (*domain.Trade, error) {
+	market, err := m.v091RepoManager.MarketRepository().GetMarketByAssets(
+		trade.MarketBaseAsset, trade.MarketQuoteAsset,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	return &domain.Trade{
+		Id:               trade.ID.String(),
+		Type:             0, // TODO
+		MarketName:       market.AccountName(),
+		MarketBaseAsset:  trade.MarketBaseAsset,
+		MarketQuoteAsset: trade.MarketQuoteAsset,
+		MarketPrice: domain.MarketPrice{
+			BasePrice:  trade.MarketPrice.BasePrice.String(),
+			QuotePrice: trade.MarketPrice.QuotePrice.String(),
+		},
+		MarketPercentageFee: domain.MarketFee{
+			BaseAsset:  0, // TODO
+			QuoteAsset: 0, // TODO
+		},
+		MarketFixedFee: domain.MarketFee{
+			BaseAsset:  uint64(trade.MarketFixedBaseFee),
+			QuoteAsset: uint64(trade.MarketFixedQuoteFee),
+		},
+		FeeAsset:     trade.MarketBaseAsset, // TODO ?
+		FeeAmount:    0,                     // TODO
```

Comment:
we have 2 options here: keeping these fields empty or calculating the fee amount always on the base asset and do populate these fields.. I think it's enough for now to leave them empty.

Reply:
Done